service/gcs/bridge/bridge.go

```diff
@@ -368,6 +369,20 @@ func (b *Bridge) createContainer(w ResponseWriter, r *Request) {
 	w.Write(response)
 }
 
+func (b *Bridge) startContainer(w ResponseWriter, r *Request) {
+	// This is just a noop, but needs to be handled so that an error isn't
```

Comment:
Can you add a `TODO: v4 impl required` so that I remember in two weeks :)

Reply:
Wait, do we want a V4 implementation? I wasn't sure if we were doing anything in start, and just didn't want it producing an error.

Reply:
Can you point me to the docs for how you envision the new protocol working? I guess I dont understand the ordering.

Reply:
"The HCS will send this message to Linux VMs in the future,...."
Do you mean RS4 or RS5?